tests/fixtures/resources/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+services:
+  caikit:
+    image: quay.io/opendatahub/caikit-tgis-serving:fast
+    volumes:
+      - ../tiny_models/models/T5ForConditionalGeneration-caikit/:/mnt/models/
```

Comment:
Unfortunately these tiny models don't seem to be working with caikit+tgis :pensive:

Reply:
Bummer

Reply:
See the updated fixtures. We can now download a model when required and test against that.